charts/topolvm/Chart.yaml

```diff
@@ -4,7 +4,7 @@ type: application
 home: https://github.com/topolvm/topolvm
 name: topolvm
 description: Topolvm
-version: 2.0.0
+version: 2.0.1
```

Comment:
I think we should release 2.0.0 after merging this PR. I don't think this version up is required.

Reply:
This change is required for the "Lint and Test Charts / lint-test" workflow to be successful.
The Helm Chart CI is validated to require incrementing the chert version compared to the main branch.

Reply:
I think this validation is effective as a countermeasure against forgetting to increment the chart version.
In this case, I think you can consider merging into main while allowing CI failures.

Reply:
Even in that case, there is no problem because the Helm chart has not been released yet.

Reply:
Yes, v2.0.0 is not released yet. So, simply release 2.0.0 with this PR.
And I think we should configure github-pages before merge this PR for Helm, is it correct?

Reply:
Since the release of Helm Chart is done manually, I think it's okay to prepare GitHub Pages after merging this Pull Request.
(refs: https://github.com/topolvm/topolvm/actions/workflows/helm-release.yaml)

This Pull Request is just prepared to rewrite deploy.md for support by Helm, not rewrite it.
TopoLVM users can still see the documentation that uses kustomize.

---

How about proceeding as follows?

1. Review and merge this Pull Request.
2. Prepare GitHub Pages
3. Release Helm Charts
4. Check Helm Chart works properly
5. Remove kustomize and rename the `README_HELM.md` to `README.md`